### PR TITLE
Fixing some formatting errors

### PR DESCRIPTION
**Pull request by MarisaJL**

### CODE REVIEW

File: topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md [hunk 1]
```diff
@@ -128,18 +130,19 @@ The first step towards reducing the effects of the cell cycle on our dataset is
...
 > 2. Rename the output `CellCycle_Annotated`
 >
-> <question-title></question-title?
+> 
+<question-title></question-title>
 >
 > 1. Why don't we need a list of genes expressed in the G1 Phase?
 >
-> > <solution-title></solution-title>
+> > <div id="solution-1" class="box-title"><button type="button" aria-controls="solution-1-contents" aria-expanded="true" aria-label="Toggle solution box: "><i class="far fa-eye" aria-hidden="true"></i><span class="visually-hidden"></span> Solution<span role="button" class="fold-unfold fa fa-minus-square"></span></button></div>
```
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md:141]: Huh, this shouldn't be necessary. Was there a bug here you were trying to fix?

File: topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md [hunk 2]
```diff
@@ -397,16 +417,15 @@ You will learn more about plotting your data in the [Filter, Plot and Explore]({
...
 > >
 > > 1. The PCA plot shows that the three groups of cells are separated out according to what phase of the cell cycle they are in. This is what we would expect to see as we are only looking at the cell cycle genes, which by definition are expressed during particular phases.
-
-> > ![PCA plot showing three separate clusters of cells in the G1, S and G2M Phases](https://github.com/MarisaJL/training-material/blob/1dd00ab33e98165d44e7275d38dd25f01e59021a/topics/single-cell/images/scrna-case_cell-cycle/CellCycle_PCA1.png)"PCA Plot of Cell Cycle Genes before regression"
-
+> >
+> > ![PCA plot showing three separate clusters of cells in the G1, S and G2M Phases](../../images/scrna-case_cell-cycle/CellCycle_PCA1.png)"PCA Plot of Cell Cycle Genes before regression"
```
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md:424]:
```suggestion
> > ![PCA plot showing three separate clusters of cells in the G1, S and G2M Phases](../../images/scrna-case_cell-cycle/CellCycle_PCA1.png "PCA Plot of Cell Cycle Genes before regression")
```

File: topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md [hunk 3]
```diff
@@ -435,16 +454,15 @@ We will now repeat the same steps to create a PCA plot of the filtered dataset a
...
 > >
 > > 1. The cells in different phases are now all mixed up together. This makes sense because we are only plotting the cell cycle genes, but the previously strong effects of the cell cycle on these genes have now been regressed out. There are still some differences between the cells (they don't all end up at the same point on the PCA chart) because the regression only removes the expected effects of the cell cycle, leaving behind any individual variation in the expression of the cell cycle genes.
-
-> > ![PCA plot showing one big cluster with the cells from G1, S and G2M Phases all mixed up together](https://github.com/MarisaJL/training-material/blob/1dd00ab33e98165d44e7275d38dd25f01e59021a/topics/single-cell/images/scrna-case_cell-cycle/CellCycle_PCA2.png)"PCA Plot of Cell Cycle Genes after regression"
-
+> >
+> > ![PCA plot showing one big cluster with the cells from G1, S and G2M Phases all mixed up together](../../images/scrna-case_cell-cycle/CellCycle_PCA2.png)"PCA Plot of Cell Cycle Genes after regression"
```
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md:461]:
```suggestion
> > ![PCA plot showing one big cluster with the cells from G1, S and G2M Phases all mixed up together](../../images/scrna-case_cell-cycle/CellCycle_PCA2.png "PCA Plot of Cell Cycle Genes after regression")
```

File: topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md (hunk 1 above)
Comment (MarisaJL) [reply topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md:141]: Some of them weren't looking right and the markdown tutorial said it should have this https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html

**Comment by hexylena** [comment]
@MarisaJL if you are done with text corrections, please leave the rest to me and I will fix the formatting issues :)

**Comment by MarisaJL** [comment]
Thank you! The text should all be OK and hopefully there's not too much for you to do now - I chose the wrong time to make this as the formatting changed between me making the tutorial files and uploading it, which added to my confusion

**Comment by hexylena** [comment]
No problem at all!

**Comment by hexylena** [comment]
Eek, that's a bug on our side. I'll look into it!

File: topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md:141]: Ahhh that's a bug on our side.

**Review by hexylena — APPROVED** [review]

**Pull request merged**